tests/unittests/algo/test_space.py

```diff
@@ -1,7 +1,412 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""Example usage and tests for :mod:`metaopt.core.io.database`."""
+"""Example usage and tests for :mod:`metaopt.algo.space`."""
 
+from collections import OrderedDict
+
+import numpy as np
+from numpy.testing import assert_array_equal as assert_eq
 import pytest
+from scipy.stats import distributions as dists
 
 from metaopt.algo.space import (Categorical, Dimension, Integer, Real, Space)
+
+
+@pytest.fixture(scope='function')
+def seed():
+    """Return a fixed ``numpy.random.RandomState`` and gloal seed."""
+    seed = 5
+    rng = np.random.RandomState(seed)
+    np.random.seed(seed)
+    return rng
+
+
+class TestDimension(object):
+    """Test methods of a Dimension object."""
+
+    def test_simple_instance(self, seed):
+        """Test Dimension.__init__."""
+        dim = Dimension('yolo', 'alpha', 0.9)
```

Comment:
Why alpha prior?

Reply:
Why not? That's a random compatible name just for the tests.

Reply:
I find it confusing. I don't know what it is.

Reply:
It was the first distribution name in `scipy.stats.distributions` when I tab-tabed in iPython :boat: 
Shall I make it 'norm'?

Reply:
Would be less confusing :)